quickstarts/completions/src/index-key.ts

```diff
@@ -0,0 +1,47 @@
+import { AzureOpenAI } from "openai";
+import { type Completion } from "openai/resources/index";
+import "dotenv/config";
+
+// You will need to set these environment variables or edit the following values
+const endpoint = process.env["AZURE_OPENAI_ENDPOINT"] || "<endpoint>";
+const apiKey = process.env["AZURE_OPENAI_API_KEY"] || "<api key>";
+
+// Azure OpenAI API version and deployment
+const apiVersion = "2024-04-01-preview";

```

Comment:
nitpick: use a new API version.
```suggestion
const apiVersion = "2024-07-01-preview";

```